packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -83,61 +83,61 @@ export const PARTITION_MAP: { [region: string]: Region } = {
 
 // https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html#access-logging-bucket-permissions
 export const ELBV2_ACCOUNTS: { [region: string]: string } = {
```

Comment:
Ordered map entries alphanumerically to facilitate scanning through it by a human.